public/app/features/templating/template_srv.ts

```diff
@@ -352,51 +351,6 @@ export class TemplateSrv implements BaseTemplateSrv {
     });
   }
 
-  getAllVariablesInTarget(target: string, scopedVars: ScopedVars, format?: string | Function): VariableMap {
-    const values: VariableMap = {};
-
-    this.replaceInVariableRegex(target, (match, var1, var2, fmt2, var3, fieldPath, fmt3) => {
-      const variableName = var1 || var2 || var3;
-      const variableDisplayName =
-        var1 || var2 || (var3 !== undefined && fieldPath !== undefined) ? `${var3}.${fieldPath}` : var3;
-      const fmt = fmt2 || fmt3 || format;
-      const value = this.getVariableValue(variableName, fieldPath, scopedVars);
-      if (value !== null && value !== undefined) {
-        const variable = this.getVariableAtIndex(variableName);
-        const text = this.getVariableText(variableName, value, scopedVars);
-        values[variableDisplayName] = this.formatValue(value, fmt, variable, text);
-      } else {
-        values[variableDisplayName] = undefined;
-      }
-
-      // Don't care about the result anyway
-      return '';
-    });
-
-    return values;
-  }
-
```

Comment:
This implementation is looks very wrong, would not work in dashboards as it only checks scopedVars and not dashboard variables.